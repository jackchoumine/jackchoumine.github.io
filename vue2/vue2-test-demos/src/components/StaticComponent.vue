<template>
  <div>
    <h1>StaticComponent</h1>
    <p>StaticComponent</p>
  </div>
</template>
