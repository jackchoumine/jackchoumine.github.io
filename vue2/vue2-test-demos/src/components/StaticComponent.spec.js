/*
 * @Author      : ZhouQiJun
 * @Date        : 2023-08-03 00:43:02
 * @LastEditors : ZhouQiJun
 * @LastEditTime: 2023-08-03 00:44:47
 * @Description :
 */
import { shallowMount } from '@vue/test-utils'
import StaticComponent from './StaticComponent.vue'
describe('StaticComponent', () => {
  it('快照测试', () => {
    const wrapper = shallowMount(StaticComponent)
    expect(wrapper.element).toMatchSnapshot()
  })
})
