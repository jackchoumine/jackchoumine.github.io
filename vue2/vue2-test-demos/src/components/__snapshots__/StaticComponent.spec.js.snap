// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StaticComponent 快照测试 1`] = `
<div>
  <h1>
    StaticComponent
  </h1>
   
  <p>
    StaticComponent
  </p>
</div>
`;
